### PR TITLE
Add Blog post for 23.07

### PR DESCRIPTION
**Pull request by sopelj**

### CODE REVIEW

File: content/blog/sprint-review-2307.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: Wazo Platform 23.07 Released
+date: 2023-05-17T19:55:00
```
Comment (fblackburn1) [inline content/blog/sprint-review-2307.md:3]: Maybe hardcode the date to Tuesday ?
Everything else LGTM :+1:

Comment (sopelj) [reply content/blog/sprint-review-2307.md:3]: Good idea. Thanks!

**Comment by sopelj** [comment]
recheck

**Comment by sopelj** [comment]
recheck

**Comment by sopelj** [comment]
recheck

**Comment by sopelj** [comment]
recheck

**Review by fblackburn1 — APPROVED** [review]

**Pull request merged**